### ISSUE TITLE
Amounts are now taxes and not actual values (before tax)

### ISSUE THREAD

**Issue by Husteln** [Geuthur/aa-ledger#32]
Hi,

I've noticed that in recent Ledger versions (using 0.5.6,  but seen it in 0.5.4 first maybe?). The money values for characters went down significantly.

I just checked wallet logs for a character and it seems that the values displayed right now, at least for Ratting, are taxes themselves. Since my corporation has a tax rate of 10, the value of money gained by character should be 9 or 10 times more (depending if you include tax payout to the corp or not).

I attached couple screenshots with wallet logs for this character - its part of a single day, and even from there its already obvious that values in Overview are corp taxes only

Is this intended, or a bug?

Thanks

![Ledger_2](https://github.com/user-attachments/assets/51ce5284-a45c-4e0e-8241-8c7b60b1aa8a)

<img width="1039" alt="Ledger_1" src="https://github.com/user-attachments/assets/9afcca79-7c5a-4aa7-9cac-9cbc99474ecc">

**Comment by Geuthur** [comment]
If you change the Tax it is not possible to display right normally a Corp don't change the tax often so.
Atm there is no system to handle this but i will add this to TODO list.

The TAX caclulcated right i tested it in many cases but if you change the TAX the amount also changed for the amounts before 

Also you need if you change the TAX to set the LEDGER_CORP_TAX:

**Comment by Geuthur** [comment]
![Screenshot 2024-08-11 093946](https://github.com/user-attachments/assets/18d4decb-4225-4044-b6e4-c1d311c5f0d8)
![Screenshot 2024-08-11 093953](https://github.com/user-attachments/assets/7f217591-93c9-470b-bf6d-f8c58cfa037a)

**Comment by Husteln** [comment]
I didn't change tax values since ledger was initially installed, so its okay there.

But I checked the code and indeed, ledger tax is only applied to ESS. In general, it should not be so. Bounty prizes coming to _corporation wallet_ are affected by the same tax values - it only receives 10% of original bounty, at least for me.

Maybe this is the reason? Bounty prizes coming from CHARACTER wallet are "value - tax", and prizes in CORPORATION wallet are "tax" only

**Comment by Geuthur** [comment]
Thats Correct the Tax is only applied to ESS cause the amount that you get from Character Ledger & Corporation is already calculated in the ESI so i not need to do so in the code.
the field  'amount' and 'tax' is same amount in any cases.

There is nothing wrong.

On the Corporation Ledger it Calculate from Corporation Journal and at Character Ledger it calcualte from Character Journal what is also correct.

Just reminder if your TAX is 10% you need to set 
```python
LEDGER_CORP_TAX: 10
```
in the local.py otherwise the ESS payout is not correct on the ledger's
the standard amount is 15% if you dont set a TAX for the ESS

explanation:

Character Journal ESI:

You only recieve the amounts from the taxed Ratting means if you character has
17.542.198 ISK - Ingame
![Screenshot 2024-08-11 133418](https://github.com/user-attachments/assets/9f753d13-202e-4f38-85a0-66333575debe)

On ESI Side it already exluded the amount you get is (missing the TAX Amount what is in this case 877109.92)
16.665.088.47 ISK - ESI Side fetch
![Screenshot 2024-08-11 133431](https://github.com/user-attachments/assets/447c2d23-f167-4f5b-a1e7-fdc1fd7873fe)
You can see it on screen that is attached i studied the ESI XD cause i have same meaning but i know now that it isn't ;)

Now we go to the Corporation Journal what is more easy cause i only need to sum the amount from bounty_prizes cause it only show the taxes already

![Screenshot 2024-08-11 134011](https://github.com/user-attachments/assets/d2f3edee-a579-4370-9faa-fb829a4b8d3e)

It only show the amount taxes so i only need to sum amounts from each char ;)

Hope that helps you to understand it ;)

**Comment by Husteln** [comment]
Again, your explanations are not in line with the numbers that I initially showed to you.

The values on first screenshot are from CORPORATION WALLET. This is ONLY 10 PERCENT of what was ratted. And this is even not a full day

Total amount of TAX received on the screenshot is ~ 4 038 980. The actual value for chat character should be at least 9 times more. not mere 15-16 million isk

<img width="1039" alt="Ledger_1" src="https://github.com/user-attachments/assets/8b8a36dc-b9cc-44e9-b677-f728f3e66cb4">

**Comment by Husteln** [comment]
So as of now, and in my case, the "total amount" is the amount of TAX paid, not INCOME

**Comment by Husteln** [comment]
Values from my first sceenshot are from _corporation wallet_, which means that it is only 10 percent (tax value) of what the character has ratted. If you calculate all the values, you will get ~ 4 038 980 ISK.

For full ratting value you multiply it by 9 or 10. You will get ~ 36 or 40 million isk. And this is not even all days of month.

Values for "total amount" are NOT correct
<img width="1039" alt="Ledger_1" src="https://github.com/user-attachments/assets/0e9e7818-30a8-4aa8-ad85-796e16b4368a">

**Comment by Husteln** [comment]
And I did all of this stuff back in 2014, so there is no need to explain it to me.

**Comment by Geuthur** [comment]
Then i don't understand what do you try to show me cause the calculation is in both ways correct.

I see a screenshot from 1 char from 10 to 7th august.

The Corporation Ledger what is on the second Screen Calculate the Complete Month but from all chars that you are registred.
i am right that you have more then 1 char registred and rat with them?

**Comment by Husteln** [comment]
If your calculation is correct, then the naming is not.

This should be named "tax paid", not "ratting amount"
![изображение](https://github.com/user-attachments/assets/bfc1cfd8-814b-4705-a652-667b151e46e2)

**Comment by Geuthur** [comment]
> If your calculation is correct, then the naming is not.
> 
> This should be named "tax paid", not "ratting amount" ![изображение](https://private-user-images.githubusercontent.com/17458608/356872638-bfc1cfd8-814b-4705-a652-667b151e46e2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzNzg4NTQsIm5iZiI6MTcyMzM3ODU1NCwicGF0aCI6Ii8xNzQ1ODYwOC8zNTY4NzI2MzgtYmZjMWNmZDgtODE0Yi00NzA1LWE2NTItNjY3YjE1MWU0NmUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODExVDEyMTU1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE1Njc0MDViNmM4MDgxNDNlNGRkZGY4MTg2MTE0M2VlMTVmMGFlODI5NTJhNjc1MmNiMGZjM2I3OTY2OTBkOTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.I7fZ39mQAqw-Gpa3tzgk0R_qzX4UIdZdAUf8Pt4QIOs)

Thats's is a point that i fully agree ^^ it can be confuse

**Comment by Geuthur** [comment]
First thanks that you giving me time by time input and issues cause this helps me a lot!

So then I notice following Issues:

Corporation Ledger TAX naming is wrong need to change from Ratting to TAX or TAX Paid
Corporation Ledger needs a Hint to understood that the Calculation is from all Chars from each Account included

**Comment by Husteln** [comment]
Yep, looks good. Thanks
